via/models/video.py

```diff
@@ -0,0 +1,12 @@
+from sqlalchemy.orm import Mapped, mapped_column
+
+from via.db import Base
+from via.models._mixins import CreatedUpdatedMixin
+
+
+class Video(CreatedUpdatedMixin, Base):
+    __tablename__ = "video"
+
+    id: Mapped[int] = mapped_column(primary_key=True, autoincrement=True, init=False)
+    video_id: Mapped[str] = mapped_column(unique=True)
+    title: Mapped[str] = mapped_column(repr=False)
```

Comment:
As we've done with transcripts (over in https://github.com/hypothesis/via/pull/1072) I've decided not to namespace videos by type/source yet (e.g. by adding a `type="youtube"` column). We can add this later if and when we ever add support for a second type of video